### ISSUE TITLE
Make alias for removed CreateHIT methods

### ISSUE THREAD

**Issue by ryantate** [ryantate/rturk#39]
In a recent version of Rturk, the RTurk::CreateHIT instance methods note=, assignments=, and auto_approval= were removed, replaced with methods with slightly different names. This affects/alters/breaks the functioning of the public RTurk::Hit.create and specifically of how new HITs are set up by users of RTurk.

Given the importance of this basic RTurk function, it would imho make sense to set up aliases pointing from the old methods to the new methods — at least until this important change could be, say, added to the Changelog with a deprecation notice. The change broke a bunch of my code and while everything is now fixed on my end it seems it would be nice to either avoid breaking code using the old public methods (however poorly named) or at least to avoid doing so without giving notice beyond the github commit log. 

It is nice to see RTurk development and improvement happening, don't get me wrong, I just wonder if we can't provide a bridge for old, working code to the shiny new world. Thanks for considering.

**Comment by mdp** [comment]
Hey Ryan,

Sorry, this was due to sloppiness on my part. I merged in a change request that should have resulted in a major change revision bump, or there should have been aliases.

At this point the best solution is probably putting in the aliases to fix for the current minor version. I'll give this a pass tonight and try and push out a fix that at least keeps the old code working properly alongside the new API.

**Comment by mdp** [comment]
Also, there's this if you're interested, or know someone who is :)

https://github.com/mdp/rturk/issues/40

**Comment by ryantate** [comment]
Thanks much Mark, and I hope my mini rants didn't drive you away, it's a great gem :)

**Comment by mdp** [comment]
No don't worry, it's a bigger issue. I basically end up working on this gem once every 6 months, usually at the behest of a pull request. It's tough to jump back into it now that I haven't used it in years. I'd like to find a maintainer or co-maintainer that's a real user of the project. There's something to be said for dogfooding.

Regardless, I believe I fixed all the breaking changes. I've added an alias_attr method to the Operations class to allow for easier aliasing. I also aliased Question to ExternalQuestion

This is all in 2.11.2

Let me know if you have any other issues
